scopes/dependencies/dependency-resolver/package-manager.ts

```diff
@@ -25,6 +25,8 @@ export type PackageManagerInstallOptions = {
   overrides?: Record<string, string>;
 
   nodeLinker?: 'hoisted' | 'isolated';
+
+  workspaceDir?: string;
```

Comment:
here we can skip the words package manager, as we are already inside the package manager.
so we can call it something like rootConfigDir, or configDir